samples/scripts/run_ci_tasks.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# run_ci_tasks.sh [OPTIONS]
+# where OPTIONS are:
+#  -a to build Android
+#  -i to build iOS
+#  -c to clean before building
+#  -b [Debug|Release]
+#  Defaults to -a -i -b Release.
+
```

Comment:
Can we remove this script now that we have the gradle stuff? what does it buy us?

Reply:
In theory it buys us consistency with the other repos I changed when running CI from our desks. `run_ci_tasks.sh -a` runs the Android CI tasks. However, it isn't consistent across all repos, and perhaps it would be better to have a separate ticket to implement a common CLI across all of our repos.